smoketest/scripts/cli/test_service_webproxy.py

```diff
@@ -297,6 +297,20 @@ def test_05_basic_squidguard(self):
         # Check for running process
         self.assertTrue(process_named_running(PROCESS_NAME))
 
+    def test_07_nocache_domain_proxy(self):
```

Comment:
Should it also have a process running check? To validate squid accepts config and starts correctly?